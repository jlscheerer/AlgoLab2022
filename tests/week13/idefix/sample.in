3
1 2 4 1
0 0
0 0
0 0
2 2 4 2
-2 0
2 0
-3 0
3 0
2 2 16 1
-2 0
2 0
-3 0
3 0
