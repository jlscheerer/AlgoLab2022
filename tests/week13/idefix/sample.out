2 0
1 16
2 4
